kplclient.go

```diff
@@ -62,6 +76,37 @@ func processChannel() {
 	}
 }
 
+func (c *KPLClient) processErrMessage() {
+	time.Sleep(time.Second * 5)
```

Comment:
If we wrap this in a `for` loop as I suggested, move this line to the end of the function.